src/styles/colors.css

```diff
@@ -1,23 +1,78 @@
 :root {
-  --palette-primary: #448fea;
-  --palette-primary-hover: #447bc6;
+  /* Action */
   --palette-action: #33ae10;
+  --palette-action-dark: #257f0c;
+  --palette-action-light: #41dd14;
   --palette-action-hover: #328a15;
-  --palette-highlight: #ff8400;
-  --palette-highlight-hover: #ffa800;
+
+  /* Caution */
   --palette-caution: #e85342;
+  --palette-caution-active: #000;
+  --palette-caution-base: #e85342;
+  --palette-caution-dark: #dc2f1b;
   --palette-caution-hover: #e53e2b;
-  --palette-warning: #ffdf8d;
+  --palette-caution-light: #ee7c6f;
+
+  /* Highlight */
+  --palette-highlight: #ff8400;
+  --palette-highlight-hover: #ffa800;
+  --palette-highlight-dark: #cc6a00;
+  --palette-highlight-light: #ff9d33;
+
+  /* Neutral */
   --palette-neutral-xx-dark: #282624;
   --palette-neutral-x-dark: #333333;
   --palette-neutral-dark: #4a4a4a;
   --palette-neutral-mid-dark: #737373;
   --palette-neutral-base: #9d9d9d;
+  --palette-neutral: #9d9d9d;
   --palette-neutral-mid: #b6b6b6;
   --palette-neutral-mid-light: #c8c8c8;
   --palette-neutral-light: #e1e1dd;
   --palette-neutral-x-light: #f5f5f2;
   --palette-neutral-xx-light: #fbfaf6;
+  --palette-neutral-hover: #000;
+
+  /* Primary */
+  --palette-primary: #448fea;
+  --palette-primary-base: #448fea;
+  --palette-primary-hover: #447bc6;
+  --palette-primary-light: #72aaef;
+  --palette-primary-dark: #1974e2;
+
+  /* Primitives */
   --white: #fff;
   --black: #000;
+
+  /* Warning */
+  --palette-warning: #ffdf8d;
+  --palette-warning-dark: #ffd15a;
+  --palette-warning-light: #ffedc0;
+  --palette-warning-hover: #ffd874;
+
+  /* Unknown -- possibly bogus */
```

Comment:
I'd search codebase for the hex values maybe, I recall these were sort of accent variations maybe used by marketing mostly